examples/scripts/ct_svmbir_ppp_bm3d_admm_prox.py

```diff
@@ -85,7 +96,7 @@
 
 
 """
-Set up an ADMM solver.
+Set up an ADMM solver with explicit NonNegativeIndicator
```

Comment:
Nitpick: missing terminating period (not only here).

Reply:
Added the missing periods